geonode/base/api/permissions.py

```diff
@@ -153,6 +154,32 @@ def has_object_permission(self, request, view, obj):
         return IsOwnerOrAdmin.has_object_permission(self, request, view, obj)
 
 
+class IsManagerEditOrAdmin(permissions.BasePermission):
+    """
+    Object-level permission to only allow admin and managers to edit a group.
+    """
+    def has_permission(self, request, view):
+        user = request.user
+        if request.method in ['POST', 'DELETE']:
+            return user and (user.is_superuser or user.is_staff)
+
+        return True
```

Comment:
In the case of None user, is wanted that this return always `True`?

Reply:
The None user is checked with `IsAuthenticatedOrReadOnly` [here](https://github.com/GeoNode/geonode/pull/8911/files#diff-0393ebb651733800057fd08d1dabbc2f4c0661b397b42810583791d213c46922R168)

Reply:
The None user is checked with `IsAuthenticatedOrReadOnly` [here](https://github.com/GeoNode/geonode/pull/8911/files#diff-0393ebb651733800057fd08d1dabbc2f4c0661b397b42810583791d213c46922R168)